internal/tools/k8s/manifest.go

```diff
@@ -0,0 +1,15 @@
+package k8s
+
+import "path"
+
+// ManifestFile is a file containing kubernetes resources.
+type ManifestFile struct {
+	Name            string
+	Directory       string
+	DeployNamespace string
+}
+
+// FQN provides the fully qualified name where the manifest is located.
```

Comment:
re: comments above, seems like it would be easier to just construct with the path and extract the base name programmatically.